static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -2,41 +2,69 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import Breadcrumbs from 'sentry/components/breadcrumbs';
-import EventOrGroupTitle from 'sentry/components/eventOrGroupTitle';
-import EventMessage from 'sentry/components/events/eventMessage';
+import Button from 'sentry/components/button';
+import Duration from 'sentry/components/duration';
 import FeatureBadge from 'sentry/components/featureBadge';
+import UserBadge from 'sentry/components/idBadge/userBadge';
 import * as Layout from 'sentry/components/layouts/thirds';
+import {KeyMetricData, KeyMetrics} from 'sentry/components/replays/keyMetrics';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import TimeSince from 'sentry/components/timeSince';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
+import {RawCrumb} from 'sentry/types/breadcrumbs';
 import {Event} from 'sentry/types/event';
-import {getMessage} from 'sentry/utils/events';
+import {generateEventSlug} from 'sentry/utils/discover/urls';
+import getUrlPathname from 'sentry/utils/getUrlPathname';
 
 type Props = {
   children: React.ReactNode;
   event: Event | undefined;
   orgId: string;
+  crumbs?: RawCrumb[];
 };
 
-function DetailLayout({children, event, orgId}: Props) {
+function DetailLayout({children, event, orgId, crumbs}: Props) {
   const title = event ? `${event.id} - Replays - ${orgId}` : `Replays - ${orgId}`;
 
   return (
     <SentryDocumentTitle title={title}>
       <React.Fragment>
         <Layout.Header>
-          <Layout.HeaderContent>
+          <HeaderContent>
             <Breadcrumbs
               crumbs={[
                 {
                   to: `/organizations/${orgId}/replays/`,
-                  label: t('Replays'),
+                  label: (
+                    <React.Fragment>
+                      {t('Replays')}
+                      <FeatureBadge type="alpha" />
+                    </React.Fragment>
+                  ),
+                },
+                {
+                  label: event
+                    ? generateEventSlug({
+                        project: event.projectSlug,
+                        id: event.id,
+                      })
+                    : t('Replay'),
                 },
-                {label: t('Replay Details')}, // TODO(replay): put replay ID or something here
               ]}
             />
+            <ButtonWrapper>
+              <Button
+                title={t('Send us feedback via email')}
+                href="mailto:replay-feedback@sentry.io?subject=Replay Details Feedback"
+              >
+                {t('Give Feedback')}
+              </Button>
```

Comment:
We could use this when the PR gets merged https://github.com/getsentry/sentry/pull/33858

Reply:
Ooooh, that's the component to use for sure.